src/test/java/org/apache/ibatis/submitted/annotion_many_one_add_resultmapid/RoleDao.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2019 ucsmy.com, All rights reserved.
+ */
```

Comment:
We cannot accept files with this header, I am afraid.
Please remove the header if that is OK with your organization.